src/routes/(auth)/login/+page.svelte

```diff
@@ -19,18 +21,30 @@
     }),
     onSubmit: async ({ email, password }) => {
       const basicAuth = createHeader(email, password);
-      const response = await fetch('/login', {
+
+      const request = await fetch('/login', {
```

Comment:
Actually this is the response

```suggestion
      const response = await fetch('/login', {
```